nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -25,6 +25,15 @@ If you want to avoid the scanning process you can explicitly configure the type
 snippet:ExplicitHostConfigType
 
 
+### Controlling assembly scanning using the command line
+
+A list of assemblies to scan can also be controlled using the `/scannedAssemblies` switch. If this option is used, the `NServiceBus.Host.exe` loads only assemblies (with references) that have been explicitly listed on the command line. Each assembly must be added using a separate switch:
```

Comment:
> A list of assemblies to scan can also be controlled using the `/scannedAssemblies` switch. If this option is used, the `NServiceBus.Host.exe` loads only assemblies (with references) that have been explicitly listed on the command line. 

What does the (with references) mean?
Does it mean that it will load all the listed assemblies + all the assemblies that the listed assembly depends on?